docs/generated/packages/eslint-plugin/documents/dependency-checks.md

```diff
@@ -8,7 +8,11 @@ We use the version numbers of the installed packages when checking whether the v
 
 ## Usage
```

Comment:
This is for the entire document. I'd have headings like

- Setup (in there mention that it should already be set up automatically, but here's how you do it manually
- Define the Build Target (or something similar; to me it wasn't obvious that the lint rule depends on a build target..like why would it. So I'd emphasize that by having a dedicated section here so ppl would immediately spot it. I'm even wondering whether the generated setup should include that rather than silently assuming `build` as the default 🤔, just to make it obvious )
- Overriding defaults (mention all the possible defaults one could override and how, basically just as you did there in the last paragraphs)